libmscore/scorediff.cpp

```diff
@@ -211,15 +211,15 @@ int MscxModeDiff::adjustSemanticsMscxOneDiff(std::vector<TextDiff>& diffs, int i
       int iScore;
       switch(diff->type) {
             case DiffType::EQUAL:
-                  /* FALLTHROUGH */
+                  /* Q_FALLTHROUGH(); */
```

Comment:
same here

Reply:
Actually here, as there are no statements between those 2 pairs of fall thouigh cases, nothing is needed, the compiler won't warn (and that's most probably the reason why those typos went unnoticed), so best remove those 2 lines entirely

Reply:
and both in this file too

Reply:
@Jojo-Schmitz so just to confirm, you're asking me to delete both `/* Q_FALLTHROUGH(); */` lines ?

Reply:
yes

Reply:
removing unneeded stuff is better than fixing it ;-)

Reply:
Indeed :) e2ca31e